src/main/java/com/locallampoon/fiveh/ui/NarrativePanel.java

```diff
@@ -33,8 +32,41 @@ public void setTextArea(String text) {
         textArea.setText(text);
     }
 
-    public void appendTextArea(String text) {
-        System.out.println(text);
-        textArea.append(text + "\n");
+    /**
+     * append text in narrative panel with customized foreground color
+     * enable and disable capability to edit before and after append
+     * @param text
+     * @param color
+     */
+    public void appendTextArea(String text, Color color)
+    {
+        this.enableNarrative();
+        this.appendToPane(this.textArea, text, color);
+        this.disableNarrative();
+    }
+
+    public void disableNarrative(){
```

Comment:
Minor detail, but I think you should rename this method to `disableNarrativeText`... and also the other to `enableNarrativeText`. It is more specific and less confusing than the existing way.

Reply:
this is totally fine. i will make changes

Reply:
I will do that tomorrow morning ... :)